### DISCUSSION TITLE
openExistingContact

### DISCUSSION THREAD

**Discussion by jibin362** [morenoh149/react-native-contacts#591] [Q&A]
I don't know if this is the right place to ask this, but seems so to me, (_First time starting a discussion._)

The thing is i want to implement "View in Address Book" just like WhatsApp. When i used **Contact.openExistingContact(contact)**. It's opening the edit form in my android phone instead of showing the contact details like WhatsApp.

_Note:
The contact data is taken from Contacts.getContactsByPhoneNumber(mobileNumber) and passed to openExistingContact_

Can someone help me out with a solution.?

**Comment by morenoh149** [comment]
openExistingContact does open the native contact editor. You probably want to call getContactsByPhoneNumber and put that data into a custom UI you build.

**Reply by jibin362** [reply]
@morenoh149 Thanks for the reply

OK. Sorry for asking this, but WhatsApp is using the default android contact app to display the contact. I wanted to implement that instead of a custom UI build into my app. If anybody have any clue that would help me out.

**Reply by morenoh149** [reply]
I don't think you would be allowed to use another app's UI from your app.

**Reply by jibin362** [reply]
There might not be a way in this package, although this one is a great one to have.

Somehow WhatsApp is managing to open contacts that way. I need to figure out how to do that.

@morenoh149 Thanks for all the replies.
